modnet/models.py

```diff
@@ -309,12 +334,12 @@ def fit(
 
         self.model.compile(
             loss=loss,
-            optimizer=keras.optimizers.Adam(lr=lr),
+            optimizer=tf.keras.optimizers.Adam(lr=lr),
             metrics=metrics,
             loss_weights=self.weights,
         )
-
-        self.history = self.model.fit(**fit_params)
+        history = self.model.fit(**fit_params)
+        self.history = history.history
```

Comment:
This might be a sensible change (as it is easier to deal with the raw dict rather than the keras history object) but it might break some of the existing benchmarking scripts

Reply:
The main advantage is that it wouldn't be pickable otherwise